Documentation/dl_build.md

```diff
@@ -2,7 +2,7 @@
 
 ## System requirements
 
-The etcd performance benchmarks run etcd on 8 vCPU, 16GB RAM, 50GB SSD GCE instances, but any relatively modern machine with low latency storage and a few gigabytes of memory should suffice for most use cases. Applications with large v2 data stores will require more memory than a large v3 data store since data is kept in anonymous memory instead of memory mapped from a file. than For running etcd on a cloud provider, we suggest at least a medium instance on AWS or a standard-1 instance on GCE.
+The etcd performance benchmarks run etcd on 8 vCPU, 16GB RAM, 50GB SSD GCE instances, but any relatively modern machine with low latency storage and a few gigabytes of memory should suffice for most use cases. Applications with large v2 data stores will require more memory than a large v3 data store since data is kept in anonymous memory instead of memory mapped from a file. For running etcd on a cloud provider, at least a medium instance on AWS or a n1-standard-1 instance on GCE is recommended. See the [Example hardware configuration][example-hardware-configurations] documentation for in-depth information on running etcd on a cloud provider.
```

Comment:
just `For running etcd on a cloud provider, see the [example hardware configuration][example-hardware-configurations] documentation.`

inlining means there's another place to track cloud configuration recommendations in addition to the recommendations already in the op-guide; would like to keep it consolidated